docs/src/configuration.md

```diff
@@ -34,46 +35,34 @@ For example:
 }
 ```
 
-**NOTE: All paths in `oranda.json` are relative to the `oranda.json` file. We
-recommend placing this file in the root of your project, alongside any other
-project manifest files you may be using, such as a `package.json` or `Cargo.toml`.
-
-## Project Metadata
-
-### Name
-
-The name set will be used in the title of your website and also as the main heading.
-
-By default Oranda will always look for this value in your `package.json`/`cargo.toml` and use it unless the value in the `oranda.json` overrides it.
-
-```json
-{
-  "name": "String"
-}
-```
-
-### Description
-
-The description set will be used in the description tag of your website and this is mostly used for SEO.
-
-By default Oranda will always look for this value in your `package.json`/`cargo.toml` and use it unless the value in the `oranda.json` overrides it.
-
-```json
-{
-  "description": "String"
-}
-```
-
-### Repository
-
-### Homepage
-
-The homepage set will be used in the meta tags of your website.
-
-By default Oranda will always look for this value in your `package.json`/`cargo.toml` and use it unless the value in the `oranda.json` overrides it.
-
-```json
-{
-  "homepage": "String"
-}
-```
+> **NOTE:** All paths in `oranda.json` are relative to the `oranda.json` file. We
+  recommend placing this file in the root of your project, alongside any other
+  project manifest files you may be using, such as a `package.json` or `Cargo.toml`.
+
+## Configuration options
+
+Some options have more detailed documentation, indicated by a link from the configuration key.
+
+- `name`: Your project's name. By default fetched from your project's manifest file.
+- `description`: Your project's description. By default fetched from your project's manifest file.
+- `dist_dir`: The directory where your static files will be output to. By default, this is `public/`.
+- `homepage`: Your project's homepage. By default fetched from your project's manifest file.
+- `static_dir`: Static content that oranda will copy to its output folder. By default, this is `static/`.
+- `no_header`: Whether to hide the page header (title and navigation bar).
```

Comment:
What a weird key name, this should really just be `header: false`?

Reply:
agreed